server/src/visitors/YmlFunctionVisitor.ts

```diff
@@ -113,7 +116,7 @@ export class YmlFunctionVisitor extends YmlBaseVisitor {
      * @returns `true` iff the provided functionName is a class method instanciation.
      */
     private isMethodInstanciation(fullName: string): boolean {
-        // At least one character and “::”.
-        return fullName.indexOf('::') > 0;
+        // Check that “::” is after the first character.
```

Comment:
I almost found the previous version clearer xD I had to look up the docs for `includes(…)` :sweat_smile:

```suggestion
        // Check that there is a “::”, with at least one character before it.
```

Reply:
> I almost found the previous version clearer xD I had to look up the docs for includes(…)

Me too. But it was treated as a code smell by Sonar :cry:

Reply:
I was mostly referring to the comments, not to the code.

Reply:
(hence the sugg)

Reply:
Any reason not to apply my sugg? xD If you don’t like it, say so xD

Reply:
Oups. Was just a miss.